fix Bug #70249:

lizhepu opened the pull request:
using getLocalTime to fix date as local time zone

jkidacki-inetsoft reviewed (commented):
This change is simple enough and doesn't pose any issues to me but it's just an extension of the changes made in the PR below. I'm concerned as to why these changes were made in the first place. They change the display time in the schedule task list to use local time zone instead of server time zone. Given that we display the current server time zone in the EM, I don't see how this makes anything easier. These types of changes should have had prior discussion before making.
https://github.com/inetsoft-technology/stylebi/pull/55

lizhepu commented:
The changes were discussed with Larry, and I have sent an email to Jason about the discussion results.

> This change is simple enough and doesn't pose any issues to me but it's just an extension of the changes made in the PR below. I'm concerned as to why these changes were made in the first place. They change the display time in the schedule task list to use local time zone instead of server time zone. Given that we display the current server time zone in the EM, I don't see how this makes anything easier. These types of changes should have had prior discussion before making. #55

Thanks for the feedback，the changes were discussed with Larry, and I have sent an email to Jason about the discussion results. And I have sent the email to you. 

Previously, we used the server timezone, but now we have switched to the local timezone because, on the cloud platform, our servers might not be located in a specific place, so using the local timezone is more appropriate. 

I have changed all places show time  to use local time zone not server time zone(portal and EM). If you find some places show server time zone, it will be a bug and we will fix it later.

jkidacki-inetsoft commented:
I wasn't aware that this was already discussed with the team. Reading through the email, I can see the justifications for these changes and I think it makes sense in that the server time zone can differ depending on where it's deployed and that it might be more useful to use local time zone instead.

This looks safe then. Please merge. Thanks.
